### PR TITLE
added helper method to transform packed items to correct location

### PR DESCRIPTION
**Pull request by SHKnudsen**

### CODE REVIEW

File: src/SpacePlanning/Generate/Packers/CuboidPacker.cs [hunk 1]
```diff
@@ -65,8 +67,8 @@ private void PackItems(List<Item> items)
             var packingResult = containerPackingResult.FirstOrDefault().AlgorithmPackingResults.FirstOrDefault();
             if (packingResult == null) throw new InvalidOperationException(PackingFailed);
 
-            // record results in this packer instance
-            this.PackedItems = CuboidsFromItems(packingResult.PackedItems);
+            // record results in this packer instance            
+            this.PackedItems = TransformPackedCuboids(this.containerCuboid, CuboidsFromItems(packingResult.PackedItems));
```
Comment (radumg) [inline src/SpacePlanning/Generate/Packers/CuboidPacker.cs]: we should separate this out on 2 lines as it's 2 distinct operations : transformation (`CuboidsFromItems`) and then moving (`TransformPackedCuboids`)

File: src/SpacePlanning/Generate/Packers/CuboidPacker.cs [hunk 2]
```diff
@@ -221,6 +223,27 @@ private static List<Item> ItemsFromCuboids(List<Cuboid> cuboids)
             return items;
         }
 
+        private static List<Cuboid> TransformPackedCuboids(Cuboid container, List<Cuboid> packedItems)
```
Comment (radumg) [inline src/SpacePlanning/Generate/Packers/CuboidPacker.cs]: name should be a bit more descriptive : `TransformPackedCuboidsToContainerCoords` maybe ?

File: src/SpacePlanning/Generate/Packers/CuboidPacker.cs (hunk 1 above)
Comment (SHKnudsen) [reply src/SpacePlanning/Generate/Packers/CuboidPacker.cs]: resolved in #134

File: src/SpacePlanning/Generate/Packers/CuboidPacker.cs (hunk 2 above)
Comment (SHKnudsen) [reply src/SpacePlanning/Generate/Packers/CuboidPacker.cs]: resolved in #134

**Pull request merged**